Yosemite/Yosemite/Actions/AnnouncementsAction.swift

```
@@ -0,0 +1,10 @@
+import Storage
+
+/// AnnouncementsAction: Defines all of the Actions supported by the AnnouncementsStore.
+///
+public enum AnnouncementsAction: Action {
+
+    /// Synchronizes the latest Announcements and save it on disk
+    ///
+    case synchronizeAnnouncements(onCompletion: (Announcement?) -> Void)
```

I think here, this should provide a `result` in the completion block to log any possible error on the network request.

---

Sounds good. I have initially thought about silently failing this request since it's not a deal breaker, but it makes sense to at lease inform developers that something bad happened! 
I will do that in a few 👍

---

Done!